package.json

```diff
@@ -253,24 +258,21 @@
     "babel-loader": "^8.1.0",
     "babel-plugin-macros": "^2.8.0",
     "babel-runtime": "^6.26.0",
-    "bootstrap": "^4.5.0",
-    "bootstrap-vue": "^2.15.0",
     "chart.js": "^2.9.3",
     "circular-dependency-plugin": "^5.2.0",
     "color": "^3.1.2",
     "concurrently": "^5.2.0",
     "css-element-queries": "^1.2.3",
     "css-loader": "^3.5.3",
     "dompurify": "^2.0.11",
-    "electron": "^6.1.12",
+    "electron": "^9.1.0",
     "electron-builder": "^22.7.0",
     "electron-notarize": "^0.3.0",
+    "electron-rebuild": "^1.11.0",
```

Comment:
> electron-rebuild not required if you use electron-builder, please consider to remove excess dependency from devDependencies

I think we should remove this.

Reply:
this is used in `yarn rebuild-pty` (this commands helps when switching electron versions)

Reply:
i'm wondering why it's caring about `devDependencies` 🤔